### PR TITLE
feat: new investment screen

### LINKED ISSUE #779: improve home screen

### PR DESCRIPTION
**Pull request by rhbrunetto**
## Changes

- Adds new layout for investment screen (cf: [Figma](https://www.figma.com/file/DtxaVafPEMzSqpdMWorQMB/Design-of-Crypto-Please-App?node-id=8479%3A2115&t=YbF7ZN0sQUuLzDcL-4))

## Related issues

Fixes #779 

## Screenshots

<details><summary>iPhone 14 Pro Max</summary>

https://user-images.githubusercontent.com/19499575/218267879-4d6f6d4c-c887-462d-9d04-6f05e7529408.mp4

Simulator Screen Shot - iPhone 14 Pro Max - 2023-02-11 at 12 46 26
Simulator Screen Shot - iPhone 14 Pro Max - 2023-02-11 at 12 46 31
Simulator Screen Shot - iPhone 14 Pro Max - 2023-02-11 at 12 49 06
Simulator Screen Shot - iPhone 14 Pro Max - 2023-02-11 at 12 50 13

</details>

<details><summary>iPhone SE 3rd Gen</summary>

Simulator Screen Shot - iPhone SE (3rd generation) - 2023-02-11 at 12 45 21
Simulator Screen Shot - iPhone SE (3rd generation) - 2023-02-11 at 12 46 35
Simulator Screen Shot - iPhone SE (3rd generation) - 2023-02-11 at 12 49 10
Simulator Screen Shot - iPhone SE (3rd generation) - 2023-02-11 at 12 50 17

</details>

<details><summary>Xiaomi MiA3</summary>

Captura de Tela 2023-02-11 às 12 47 24
Captura de Tela 2023-02-11 às 12 47 32
Captura de Tela 2023-02-11 às 12 49 20
Captura de Tela 2023-02-11 às 12 50 11

</details>

## Checklist

- [x] PR is ready for review (if not, it should be a draft).
- [x] PR title follows [Conventional Commits][1] guidelines.
- [x] Screenshots/video added.
- [ ] Tests added.
- [x] Self-review done.

[1]: https://www.conventionalcommits.org/en/v1.0.0/

### CODE REVIEW

**Comment by ookami-kb** [comment]
@rhbrunetto please fix conflicts.

File: packages/espressocash_app/pubspec.lock [hunk 1]
```diff
@@ -1934,5 +1934,5 @@ packages:
     source: hosted
     version: "3.1.1"
 sdks:
-  dart: ">=2.19.0 <3.0.0"
+  dart: ">=2.19.0 <4.0.0"
```
Comment (ookami-kb) [inline packages/espressocash_app/pubspec.lock:1937]: Not needed.

File: packages/espressocash_app/lib/app/screens/authenticated/investments/components/investment_header.dart [hunk 2]
```diff
@@ -10,150 +12,248 @@ import '../../../../../features/ramp/widgets/ramp_buttons.dart';
...
   const InvestmentHeader({super.key});
 
   @override
-  Widget build(BuildContext context) => DecoratedBox(
-        decoration: const BoxDecoration(
-          color: Color(0xff2D2B2C),
-          borderRadius: BorderRadius.all(Radius.circular(5)),
-        ),
+  Widget build(BuildContext context) => Container(
+        height: 280,
```
Comment (ookami-kb) [inline packages/espressocash_app/lib/app/screens/authenticated/investments/components/investment_header.dart:26]: I don't like this hard-coded height. Can we do it in a different way?

File: packages/espressocash_app/lib/l10n/app_en.arb [hunk 3]
```diff
@@ -526,5 +527,10 @@
       }
     }
   },
-  "invalidEscrowError" : "The link is incorrect or the money has already been withdrawn."
+  "invalidEscrowError" : "The link is incorrect or the money has already been withdrawn.",
+  "fundYourAccount": "Fund your account to begin your crypto journey. You can buy and sell crypto or send money anywhere in the world!",
+  "investmentHeaderButtonsTitle": "I want to",
+  "sendMoney": "Send Money",
+  "startedInvestingTitle": "Started Investing",
```
Comment (ookami-kb) [inline packages/espressocash_app/lib/l10n/app_en.arb:534]: Shouldn't it be `Start Investing`?

**Review by ookami-kb — CHANGES_REQUESTED** [review]

File: packages/espressocash_app/lib/l10n/app_en.arb (hunk 3 above)
Comment (antoineherzog) [reply packages/espressocash_app/lib/l10n/app_en.arb:534]: Yes I think so!

File: packages/espressocash_app/pubspec.lock (hunk 1 above)
Comment (ookami-kb) [reply packages/espressocash_app/pubspec.lock:1937]: Not resolved.

**Review by ookami-kb — CHANGES_REQUESTED** [review]

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**